Unix/pal/pal

```diff
@@ -210,11 +210,7 @@
 **==============================================================================

```

Comment:
This looks like an old version of palcommon.h.    What happens if you delete this file?